views/mail/suspend.htm

```diff
@@ -0,0 +1,12 @@
+subject = "Account {{ name }} {{ surname }} has been suspended."
+description = "The system has suspended an account"
+==
+The following user's account has been suspended.
+
+This normally happens when the user attempts to log in with the wrong password too many times:
```

Comment:
Should this include the user's IP address since account suspension is tied to IP address isn't it?

Reply:
It could, yes. May assist in detecting distributed brute force attacks as well

Reply:
Are you able to add that to the PR?